.github/workflows/coverity.yml

```diff
@@ -12,12 +16,18 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
       - name: Prepare environment
+        env:
+          DEBIAN_FRONTEND: 'noninteractive'
         run: |
-          ./packaging/installer/install-required-packages.sh --dont-wait --non-interactive netdata
-          sudo apt-get install -y libjson-c-dev libipmimonitoring-dev libcups2-dev libsnappy-dev \
-                                  libprotobuf-dev libprotoc-dev libssl-dev protobuf-compiler
+          ./packaging/installer/install-required-packages.sh \
+            --dont-wait --non-interactive netdata
+          sudo apt-get install -y libjson-c-dev libipmimonitoring-dev \
+                                  libcups2-dev libsnappy-dev libprotobuf-dev \
+                                  libprotoc-dev libssl-dev protobuf-compiler \
+                                  libnetfilter-acct-dev
```

Comment:
`netdata-all` should be used so that we don't need an extra install step. If it doesn't pull in these packages, that's a bug and should be handled as such.

Reply:
Whilst I agree in general, these however are "dev" dependencies and probably optional where normally a user would not need to install them. However as pointed out in our sync/meeting just now we probably are legitimately missing some things in our `install-required-pacakges.sh` script :/ We should fix that; but let's file a separate issue and do it there so we can merge this PR :)

Reply:
Please link/ref the issue here once you've created it :)